content/manuals/engine/network/ca-certs.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Use CA certificates with Docker
+linkTitle: CA certificates
+description: Learn how to install and use CA certificates on the Docker host and in Linux containers
+keywords: docker, networking, ca, certs, host, container, proxy
+---
+
+If your company uses a proxy that inspects HTTPS traffic, you might need to add
+your company’s root certificates to both your host machine and your Docker
```

Comment:
> need to add your company’s root certificates

may be we should change it to "proxy's root certificates" or  "the required root certificates to both..."